extension/options.html

```diff
@@ -1,155 +1,209 @@
+              </tr>
+              <tr>
+                <td>C</td>
+                <td>Copy to clipboard</td>
+              </tr>
+              <tr>
+                <td>D</td>
+                <td>Hide/show definitions</td>
+              </tr>
+              <tr>
+                <td>Shift/Enter&nbsp;&nbsp;</td>
+                <td>Switch dictionaries</td>
+              </tr>
+              <tr>
+                <td>B</td>
+                <td>Previous character</td>
+              </tr>
+              <tr>
+                <td>M</td>
+                <td>Next character</td>
+              </tr>
+              <tr>
+                <td>N</td>
+                <td>Next word</td>
+              </tr>
+              <tr>
+                <td>J</td>
+                <td>Scroll back definitions</td>
+              </tr>
+              <tr>
+                <td>K</td>
+                <td>Scroll forward definitions</td>
+              </tr>
+            </table>
+            <br />
+            <input type="checkbox" id="disablekeys" name="disablekeys" />
+            Disable these keys
+          </div>
+          <div id="kanjicon" class="tabContent">
+            <h1>Kanji Dictionary</h1>
+            <p>
+              <strong>Displayed information:</strong><br /><br />
+              <input
+                type="checkbox"
+                id="kanjicomponents"
+                name="kanjicomponents"
+              />
+              Kanji Components
+              <br />
+              <input type="checkbox" id="H" name="H" />
+              Halpern
+              <br />
+              <input type="checkbox" id="L" name="L" />
+              Heisig 5th Edition
+              <br />
+              <input type="checkbox" id="LF" name="LF" />
+              Heisig 6th Edition
+              <br />
+              <input type="checkbox" id="E" name="E" />
+              Henshall
+              <br />
+              <input type="checkbox" id="DK" name="DK" />
+              Kanji Learners Dictionary
+              <br />
+              <input type="checkbox" id="N" name="N" />
+              Nelson
+              <br />
+              <input type="checkbox" id="V" name="V" />
+              New Nelson
+              <br />
+              <input type="checkbox" id="Y" name="Y" />
+              PinYin
+              <br />
+              <input type="checkbox" id="P" name="P" />
+              Skip Pattern
+              <br />
+              <input type="checkbox" id="IN" name="IN" />
+              Tuttle Kanji &amp; Kana
+              <br />
+              <input type="checkbox" id="I" name="I" />
+              Tuttle Kanji Dictionary
+              <br />
+              <input type="checkbox" id="U" name="U" />
+              Unicode
+              <br />
+            </p>
+          </div>
+          <div id="clipcon" class="tabContent">
+            <h1>Copy to Clipboard</h1>
+            <p></p>
```

Comment:
Tested in chrome and found that removing this doesn't change spacing at all so will delete.